constants/labels.js

```diff
@@ -0,0 +1,64 @@
+
+const prefixes = {
+    CUSTOMER: 'Customer',
+    BUG: 'Bug',
+    FEATURE: 'Feature'
+};
+
+const bugTypes = {
+    PRODUCTION: 'production',
+    DEVELOPMENT: 'development'
+}
+
+const featureTypes = {
+    NEW: 'new',
+    EXISTING: 'existing'
+}
+
+const getPrefix = (label) => (label || '').split(':')[0].trimRight();
+
+function getType (label) {
+    const labelType = (label || '').split(':');
+    return (labelType.length > 1) ? labelType[1].trimLeft() : '';
+}
+
+function getTitle (label) {
+    const prefix = getPrefix(label);
+    const type = getType(label);
+
+    switch(prefix) {
+        case prefixes.FEATURE:
+            switch (type) {
+                case featureTypes.EXISTING:
+                    return `\n\n## Improvements`;
```

Comment:
Don't think we should return formatting in the string. Leave formatting to the caller IMO. E.g.:

```suggestion
                    return `Improvements`;
```

Reply:
I'll be doing this and the next one on a single commit.